engines/tinsel/dialogs.h

```diff
@@ -352,6 +352,9 @@ class Dialogs {
 	const FILM *GetWindowData();
 	void Redraw();
 
+	// noir
```

Comment:
In the interest of grep-ability, it would be useful to stick to one capitalization of these.